### PR TITLE
Update DPE spec

### PR DESCRIPTION
**Pull request by jhand2**
* Remove FIPS-mode derivation. This is not necessary given updated guidance on deterministic key derivation.
* Add list of supported DPE commands.
* Remove reliance on PCRs.

### CODE REVIEW

File: runtime/README.md [hunk 1]
```diff
@@ -334,38 +356,13 @@ Caliptra Runtime firmware is responsible for initializing DPE’s Default Contex
...
-These PCRs will store only the cumulative journey of the PCRs.
-
-#### PCR Properties
-
-* 48 bytes
-* Initial value of all zeros
-* Extend operation: `NEW_VALUE = SHA384_HASH(OLD_VALUE || MEASUREMENT)`
-* Clear operation: Resettable to initial value by Runtime Firmware
-
-#### DPE PCR Usage
```
Comment (pkwidzin-amd) [inline runtime/README.md:354]: There are some other references to PCR (eg. table below) that should be rephrase - can scrub the rest of spec for those?

File: runtime/README.md [hunk 2]
```diff
@@ -381,69 +378,21 @@ DPE will first collect measurements and concatenate them in a byte buffer
...
-  * Derive an ECC P384 keypair from KV slot 0 CDI into the next available Runtime
-    KeyVault Slot
-  * If no slots are available, evict the oldest Runtime slot and use that slot.
-* For CertifyKey: Request the public key
-* For Sign: Sign passed data
-* Erase KeyVault slot 0
-
-### FIPS Mode Caching
-
-Caliptra has 32 KeyVault slots and a max of 24 DPE contexts. To accomodate
```
Comment (pkwidzin-amd) [inline runtime/README.md:423]: Is that part FIPS specific? I thought regardless of FIPS we still have those upper limits?

Comment (pkwidzin-amd) [reply runtime/README.md:423]: I guess this be derived implicitly from max TCI nodes so maybe add a statement in that section that max contexts == max TCI nodes

Comment (jhand2) [reply runtime/README.md:423]: In the deterministic (regular) mode, we can derive a key, use it, and then clear the slot. So number of hw KeyVault slots isn't a limitation.

File: runtime/README.md (hunk 1 above)
Comment (jhand2) [reply runtime/README.md:354]: Removed (note there are still some PCR references which refer to PCRs that hold measurements of Caliptra firmware)

Comment (pkwidzin-amd) [reply runtime/README.md:354]: Thanks

**Review by rusty1968 — APPROVED** [review]

**Pull request merged**